alacritty/src/window.rs

```diff
@@ -449,8 +467,8 @@ fn x_embed_window(window: &GlutinWindow, parent_id: c_ulong) {
     }
 }
 
-#[cfg(not(any(target_os = "macos", windows)))]
+#[cfg(all(feature = "x11", not(any(target_os = "macos", windows))))]
 unsafe extern "C" fn xembed_error_handler(_: *mut XDisplay, _: *mut XErrorEvent) -> i32 {
-    error!("Could not embed into specified window.");
+    log::error!("Could not embed into specified window.");
```

Comment:
This is kinda inconsistent with every other place where we log. I'd assume you're trying to save yourself the cfg'ed import?

It's probably okay right here, we do similar stuff for `exit`.

Reply:
I can bring back old annotations should be that complex, if you want them.

Reply:
I think this is fine, just wanted to make sure.